theme/save-playgrounds.js

```diff
@@ -0,0 +1,46 @@
+(function savePlaygrounds() {
+  function setCodeToPlayground() {
+    var codes = JSON.parse(
+      localStorage.getItem(`${window.location.href}₹code`)
+    );
+    if (codes) {
+      var i = 0;
+      Array.from(document.querySelectorAll(".playground")).forEach(function (
+        pre_block
+      ) {
+        let code_block = pre_block.querySelector("code");
+        let editor = window.ace.edit(code_block);
+        editor.setValue(codes[i]);
+        editor.clearSelection();
+        i += 1;
+      });
+    }
+  }
+  function getCodeFromPlayground() {
+    var codes = [];
+    Array.from(document.querySelectorAll(".playground")).forEach(function (
+      pre_block
+    ) {
+      let code_block = pre_block.querySelector("code");
+      let editor = window.ace.edit(code_block);
+      let code = editor.getValue();
+      codes.push(code);
+    });
+    localStorage.setItem(`${window.location.href}₹code`, JSON.stringify(codes));
```

Comment:
Has it already been discussed what the `₹` character is doing here? It looks a bit strange to me :smile: 

Have you considered using just the [`pathname`](https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname) instead of the full URL? Since local storage is different for different hosts, I don't think we need the full URL here.

Reply:
Oh, if the `₹` is necessary somehow, then I would suggest creating a new function to construct the key. You can then document the use there — and also call it from both places that need this key.

Reply:
> Oh, if the `₹` is necessary somehow, then I would suggest creating a new function to construct the key. You can then document the use there — and also call it from both places that need this key.

Sorry, I didn't understand creating of "another function for key creating".